### ISSUE TITLE
Ch15 Alt Text

### ISSUE THREAD

**Issue by pancakereport** [data-8/textbook#235]
- [X] 15.0 heights.scatter('MidParent')
- [X] 15.0 heights_with_predictions.scatter('MidParent')
- [X] 15.1 hybrid.scatter('acceleration', 'msrp')
- [X] 15.1 hybrid.scatter('mpg', 'msrp')
- [X]  15.1 suv.scatter('mpg', 'msrp')
- [X] 15.1 suv.scatter('acceleration', 'msrp')
- [X] 15.1 plots.ylim(-3, 3);
- [X] 15.1 plots.ylim(-3, 3);
- [X] 15.1 r_scatter(0.9)
- [X] 15.1 r_scatter(0.25)
- [X] 15.1 r_scatter(0)
- [X] 15.1 r_scatter(-0.55)
- [X] 15.1 t.scatter(0, 1, s=30, color='red')
- [X] 15.1 t.scatter('y', 'x', s=30, color='red')
- [X] 15.1 nonlinear.scatter('x', 'y', s=30, color='r')
- [X] 15.1 line.scatter('x', 'y', s=30, color='r')
- [X] 15.1 outlier.scatter('x', 'y', s=30, color='r')
- [X] 15.1 sat2014.scatter('Critical Reading', 'Math')
- [X] 15.1 The way the image after "The following graph, provided in the paper, should motivate you to go and take a look." is embedded with Python (I think) and doesn't have any good alt text
- [X] 15.2 heights_with_predictions.scatter('MidParent')
- [X] 15.2 heights_with_su_predictions.scatter('MidParent SU')
- [X] 15.2 matplotlib after "The 45 degree line is shown in red."
- [X] 15.2 matplotlib after "The red line is too high to pick off the center."
- [X] 15.2 matplotlib after "That line is the green one shown below."
- [X] 15.2 regression_line(0.95)
- [X] 15.2 regression_line(0.6)
- [X] 15.2 heights_with_predictions.scatter('MidParent')
- [X] 15.2 heights.with_column('Fitted', fit(heights, 'MidParent', 'Child')).scatter('MidParent')
- [X] 15.2 heights.scatter('MidParent', fit_line=True)
- [X] 15.2 baby.scatter('Maternal Height', 'Maternal Pregnancy Weight', fit_line=True)
- [X] 15.3 little_women.scatter('Periods', 'Characters')
- [X] 15.3 lw_with_predictions.scatter('Periods')
- [X] 15.3 lw_errors(lw_reg_slope, lw_reg_intercept)
- [X] 15.3 lw_errors(50, 10000)
- [X] 15.3 lw_errors(-100, 50000)
- [X] 15.3 lw_rmse(50, 10000)
- [X] 15.3 lw_rmse(-100, 50000)
- [X] 15.3 lw_rmse(90, 4000)
- [X] 15.3 lw_rmse(lw_reg_slope, lw_reg_intercept)
- [X] 15.3 lw_rmse(lw_reg_slope, lw_reg_intercept)
- [X] 15.4 shotput.scatter('Weight Lifted')
- [X] 15.4 shotput.with_column('Best Straight Line', fitted).scatter('Weight Lifted')
- [X] 15.4 shotput.with_column('Best Quadratic Curve', shotput_fit).scatter(0)
- [X] 15.5 scatter_fit(heights, 'MidParent', 'Child')
- [X] 15.5 residual_plot(heights, 'MidParent', 'Child')
- [X] 15.5 regression_diagnostic_plots(heights, 'MidParent', 'Child')
- [X] 15.5 regression_diagnostic_plots(dugong, 'Length', 'Age')
- [X] 15.5 regression_diagnostic_plots(hybrid, 'acceleration', 'mpg')
- [X] 15.6 scatter_fit(heights, 'MidParent', 'Child')
